### PR TITLE
Non functional requirements

### PR DESCRIPTION
**Pull request by Prolas**
Added non functional requirements

### CODE REVIEW

**Review by mathieumognol — APPROVED** [review]
LGTM but maybe we could mention somewhere that first time the user access the flight screen we request location permission

File: template/6-non_functional_requirements.md [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+* The application should primarily function as a practical tool and must not distract or jeopardize the safety of those on board the balloon.
+* Adoption metrics must focus on user engagement quality rather than frequency, emphasizing the app's value-add to the users' experience.
+
+### Scalability
+
+Scalability
+
+The application must be designed to handle varying user loads effectively, particularly during peak periods such as peak tourist seasons, large group bookings, or popular local events. This ensures that balloon company operations remain smooth and efficient even under high demand.
+
+* Automatic Scaling: The application should dynamically allocate resources based on real-time user load, ensuring consistent performance.
```
Comment (Plouc314) [inline template/6-non_functional_requirements.md]: As the application will be purely frontend based, we won't have servers, instead we will use Firebase products, so we won't allocate resources directly, Firebase will. That means that we actually do not focus on the scalability of the traffic/workload on the app, but rather scalability in terms of clients (which may not be relevant in this section, that I'm not sure)

File: template/6-non_functional_requirements.md [hunk 2]
```diff
@@ -1,16 +1,46 @@
...
+
+* Optimal Performance: Maintain fast response times and efficient operation, even during peak usage periods such as weekends, holidays, and local events.
 
-*Are there known periods of bursty traffic that the MVP must be designed to support?*
+### Availability
 
+* The application must maintain a minimum uptime of 99.9% to ensure the company remains operational and is not hindered by an unavailable tool.
+* Implement a highly available infrastructure to minimize downtime. This includes using redundant servers, failover mechanisms, and geographically distributed data centers to ensure continuous operation even if one component fails.
```
Comment (Plouc314) [inline template/6-non_functional_requirements.md]: As above, we won't have servers (except for the monitoring), so our availability is exclusively dependent on Firebase availability, which is (from what I [found](https://firebase.google.com/terms/service-level-agreement)) around 99.95%, so we can keep the 99.9%.

File: template/6-non_functional_requirements.md (hunk 1 above)
Comment (Prolas) [reply template/6-non_functional_requirements.md]: I will change it

File: template/6-non_functional_requirements.md (hunk 2 above)
Comment (Prolas) [reply template/6-non_functional_requirements.md]: I will change it

File: template/6-non_functional_requirements.md [hunk 3]
```diff
@@ -33,16 +33,12 @@ Users associated with the same flight have access to a chat where they can poten
 
 ### Scalability
 
-Scalability
-
 The application must be designed to handle varying user loads effectively, particularly during peak periods such as peak tourist seasons, large group bookings, or popular local events. This ensures that balloon company operations remain smooth and efficient even under high demand.
 
-* Automatic Scaling: The application should dynamically allocate resources based on real-time user load, ensuring consistent performance.
+The application integrates automated deployment for new clients, including Firebase provisioning, database setup (rules, index createion...), and monitoring service configuration.
```
Comment (Plouc314) [inline template/6-non_functional_requirements.md]: Typo in creation

**Pull request merged**